lib/rubocop/cop/rake/class_definition_in_namespace.rb

```diff
@@ -3,13 +3,13 @@
 module RuboCop
   module Cop
     module Rake
-      # Detects class or module definition in a task or namespace,
+      # Detects class or module definition in a namespace,
       # because it is defined to the top level.
-      # It is confusing because the scope looks in the task or namespace,
+      # It is confusing because the scope looks in the namespace,
```

Comment:
@kuahyeow This grammatical issue still needs to be fixed. It is incorrect to say it "the scope looks in the namespace", which may imply to some that the namespace is relevant to the scoping. It makes more sense to say it "appears as if it is in the namespace"... when in fact it is not.

```suggestion
      # It is confusing because the scope appears as if it is in the namespace,
```